geode-core/src/main/java/org/apache/geode/management/internal/web/controllers/RegionManagementController.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.management.internal.web.controllers;
+
+import static org.apache.geode.management.internal.web.controllers.AbstractAdminRestController.MANAGEMENT_API_VERSION;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import org.apache.geode.cache.configuration.RegionConfig;
+import org.apache.geode.security.ResourcePermission;
+
+@Controller("regionManagement")
+@RequestMapping(MANAGEMENT_API_VERSION)
+public class RegionManagementController extends AbstractManagementController {
+
+  @RequestMapping(method = RequestMethod.POST, value = "/regions")
+  public ResponseEntity<String> createRegion(@RequestBody RegionConfig regionConfig) {
+    securityService.authorize(ResourcePermission.Resource.DATA,
+        ResourcePermission.Operation.MANAGE);
+    return new ResponseEntity(regionConfig.getName(), HttpStatus.CREATED);
+  }
```

Comment:
I think it might be weird to give CREATED status when we haven't actually created anything?

Reply:
this is just a dummy implementation, it's returnning CREATED so that later on we don't have to change the test expected status